sdk/core/azure-core/inc/azure/core/logging/logging.hpp

```diff
@@ -3,7 +3,7 @@
 
 #pragma once
 
-#include "azure.hpp"
+#include "azure/core/azure.hpp"
```

Comment:
Unrelated to this PR, why do we have a header file named `azure.hpp`. We should rename it.

Reply:
At one point there was a desire to have a top level header.  The desire was to have one header to include all services present, i.e. `#include "azure/azure.hpp"` and get everything azure.  Similarly with `#include "storage/blobs.hpp"` to get storage blobs.